ocaml-lsp-server/docs/ocamllsp/inferIntf-spec.md

```diff
@@ -0,0 +1,27 @@
+#### Infer Interface Request
+
+Infer Interface Request is sent from the client to the server to get the infered
```

Comment:
The docs are good, but let's make sure to mention the most important thing: these custom requests aren't to be relied on and can be removed without warning.